src/vs/workbench/services/environment/browser/environmentService.ts

```diff
@@ -18,13 +18,16 @@ import { parseLineAndColumnAware } from 'vs/base/common/extpath';
 import { LogLevelToString } from 'vs/platform/log/common/log';
 import { ExtensionKind } from 'vs/platform/extensions/common/extensions';
 import { isUndefined } from 'vs/base/common/types';
+import { CommandsRegistry } from 'vs/platform/commands/common/commands';
 
 class BrowserWorkbenchConfiguration implements IWindowConfiguration {
 
 	constructor(
 		private readonly options: IBrowserWorkbenchOptions,
 		private readonly payload: Map<string, string> | undefined
-	) { }
+	) {
+		CommandsRegistry.registerCommand('workbench.getTarballProxyEndpoints', () => (this.options.tarballProxyEndpoints ?? {}));
```

Comment:
I would rather register the command in `workbench.web.api.ts` where also the API exists, maybe from the `create` method. We already use `CommandRegistry` there.

Reply:
Also should it rather be called `_workbench.getTarballProxyEndpoints`?